canonical-web-and-design/ubuntu.com
# Update discourse topics to original topics

## Done

- Updated all discourse docs index topics to use updated version
- Revert back to the original topics

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Make sure all sets of docs work as expected
